gnosis/safe-react
# Hotfix: fix upgrade functionality

**This PR**:
- Fixes upgrade by changing the order of transactions (mastercopy update first, then the fallback handler setup)
- Rename `updateSafe.js` file which contained function for updating safe to avoid naming confusion with `updateSafe.js` redux action

**How to test**:
1. Create a safe with 1.0.0 contracts
2. Load and upgrade the safe from the interface
3. Send a POST request to `rinkeby.infura.io/v3/c5702fc604614f3aadb151ee5ea0e709` with the following body:
```
{
	"jsonrpc": "2.0",
	"id": 1,
	"method": "eth_getStorageAt",
	"params": [your safe address, "0x6c9a6c4a39284e37ed1cf53d337577d14212a4870fb976a4366c693b939918d5", "latest"]
}

Second string in `params` array is a position of fallback manager address taken from here -->
https://github.com/gnosis/safe-contracts/blob/development/contracts/base/FallbackManager.sol#L10
```
4. The response should look like this
```
{
    "jsonrpc": "2.0",
    "id": 1,
    "result": "0x000000000000000000000000d5d82b6addc9027b22dca772aa68d5d74cdbdf44"
}
```
the address in result should equal to the address of the fallback handler (`DEFAULT_FALLBACK_HANDLER_ADDRESS`)
